src/DotLiquid.Tests/TruthyAndFalsyTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using NUnit.Framework;
+
+namespace DotLiquid.Tests
+{
+
+    [TestFixture]
+    public class TruthyAndFalsyTests
```

Comment:
Should you have the specification sample templates as tests also here?

i.e. https://shopify.github.io/liquid/basics/truthy-and-falsy/

Reply:
Done